src/aiod/authorisation/authorisation.py

```diff
@@ -0,0 +1,80 @@
+import os
```

Comment:
Nitpick, but I think this is _authentication_ and not _authorization_. And I think `get_current_user` can be moved to this file.